tests/Integration/Behaviour/Features/Context/Domain/AliasFeatureContext.php

```diff
@@ -89,6 +92,29 @@ public function assertAliasesList(TableNode $table): void
         }
     }
 
+    /**
+     * @When I toggle alias with following information:
+     *
+     * @param TableNode $table
+     *
+     * @throws AliasConstraintException
+     */
+    public function toggleAlias(TableNode $table): void
+    {
+        $aliasReferences = $table->getColumnsHash();
+
+        foreach ($aliasReferences as $aliasReference) {
+            /** @var string $aliasId */
+            $aliasId = $this->getSharedStorage()->get($aliasReference['id reference']);
+
+            try {
+                $this->getCommandBus()->handle(new ToggleAliasCommand(new AliasId((int) $aliasId)));
+            } catch (InvalidArgumentException $exception) {
+                $this->setLastException($exception);
+            }
+        }
```

Comment:
```suggestion
    /**
     * @When /^I (enable|disable) alias with reference "(.+)"$/
     *
     * @param bool $enable
     * @param string $aliasReference
     *
     * @see StringToBoolTransformContext::transformTruthyStringToBoolean for $enable string to bool transformation
     */
    public function toggleAliasStatus(bool $enable, string $aliasReference): void
    {
        $this->getCommandBus()->handle(
            new UpdateAliasStatusCommand($this->getSharedStorage()->get($aliasReference), $enable)
        );
    }
```

Reply:
and don't forget to include the StringToBoolTransformContext into contexts list of alias suite in behat.yml for transformation to take effect